prompting-client/src/daemon/server.rs

```diff
@@ -165,6 +209,21 @@ where
             "this endpoint is not yet implemented",
         ))
     }
+
+    async fn set_logging_level(
+        &self,
+        level: Request<String>,
+    ) -> Result<Response<SetLoggingLevelResponse>, Status> {
+        let current = log_filter(&level.into_inner());
+
+        match self.reload_handle.set_filter(&current) {
+            Ok(_) => Ok(Response::new(SetLoggingLevelResponse { current })),
+            Err(e) => Err(Status::new(
+                Code::InvalidArgument,
+                format!("unable to set logging level: {e}"),
+            )),
+        }
+    }
```

Comment:
Here `level` is passed into `log_filter` as `filter`, but I don't think of levels and filters as really the same thing. What can `level` be set to, and is it validated anywhere?